### PR TITLE
translated adding page translations.md

### PR DESCRIPTION
**Pull request by KillMonger1**

### CODE REVIEW

File: docs/docs/adding-page-transitions.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Adding Page Transitions
+title: पेज ट्रांज़िशन ऐड करना 
 overview: true
 ---
 
-Page transitions give users a better experience when navigating between pages. There are a number of ways to add them to your Gatsby site.
+पेज ट्रांज़िशन उपयोगकर्ताओं को पेजेस के बीच नेविगेट करते समय बेहतर अनुभव देते है। उन्हें अपने Gatsby साइट पर ऐड के कई तरीके हैं।
```
Comment (arshadkazmi42) [inline docs/docs/adding-page-transitions.md]: `पर ऐड के` => `पर ऐड करने के`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Looks good. 
One small fix and this is good to go

**Comment by KillMonger1** [comment]
Done

**Review by arshadkazmi42 — APPROVED** [review]
Great work 🎉

**Pull request merged**